lib/aio/job.py

```diff
@@ -151,6 +151,7 @@ async def run_container(job: Job, subject: Subject, ctx: JobContext, log: LogStr
 
             finally:
                 await run([*ctx.container_cmd, 'rm', '--force', '--time=0', f'--cidfile={tmpdir}/cidfile'],
+                          stderr=asyncio.subprocess.STDOUT,
```

Comment:
This *hides* stderr (as stdout goes to /dev/null). The default is that stderr goes to the terminal or log file, i.e. you should currently see stderr in `podman logs`.